### PR TITLE
Ports Horrorform (WIP, DNM, etc)

### PR DESCRIPTION
**Pull request by ShadowDeath6**
Ports https://github.com/yogstation13/yogstation-old/pull/1132

It's MOSTLY completed outside of a really weird bug which I'll get to later

Also adds: 
Screech stunning borgs
Fleshmend heals toxin damage
Screech breaks lights
Hulk not doing damage to every body part

Might be more than one bug, but the main one is that for some reason the changeling doesn't get the correct sprite upon reversion to human form.
Step 1. Be horrorform, transformation works completely fine
Step 2. Revert in any way, it doesn't matter
Step 3. Have skin be completely invisible, only have hair
I tried it as a golem ling and reversion worked fine but I had no ling armor, most likely because golems can't wear most clothing. Reverted back into a human correctly.

:cl: ShadowDeath6
rscadd: Nanotrasen scientists find that the mysterious entities known as Changelings appear to be exhibiting signs of rapid genetic distortion. Previous reports show that this could be hazardous to the workplace. Remember to check fellow crewmates for signs of inhuman behavior.
tweak: Hulk no longer penetrates all armor and hits every limb.
/:cl:

### CODE REVIEW

**Comment by Stealthkibbler** [comment]
Anatomic panacea is for healing toxins, fleshmend is just for burn and brute.

**Comment by ShadowDeath6** [comment]
horrorform can't use ling abilities which means they'll have no way to deal with toxin damage. I would have it only purge chems but morphining an abomination should still be an option.

**Comment by Stealthkibbler** [comment]
Ahk

**Comment by GrayRachnid** [comment]
> Screech stunning borgs

there is another Screech specifically made for this, the EMP screech. Making the deafening screech stun humans as well as borgs is overpowered. Source: Abused it myself.

**Comment by ShadowDeath6** [comment]
@GrayRachnid 
resonant screech stuns borgs
also horrorform can't use regular ling abilities

**Comment by GrayRachnid** [comment]
Just read the code, ignore what I said.

**Comment by FluffySurvivor** [comment]
Default armblade in horror mode pls

**Comment by ShadowDeath6** [comment]
horrorform has stronger melee than armblade, what would that accomplish

**Comment by FluffySurvivor** [comment]
theme and good xeno feeling 

just make the horroform armblade as strong as your current horrorform punches

File: code/game/gamemodes/changeling/powers/horrorform.dm [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+		var/newNameId = "Unknown"
+		user.real_name = newNameId
+		for(var/obj/item/I in user) // removes any item, the only thing I can think of is cuffs
+			user.unEquip(I)
+		for(var/obj/item/I in user) // removes the abomination armor
+			qdel(I)
+		changeling.chem_recharge_slowdown = 0
+		var/mob/living/carbon/human/H = user
+		H.set_species(/datum/species/human)
+		H.update_body()
```
Comment (Super3222) [inline code/game/gamemodes/changeling/powers/horrorform.dm]: Update body parts() covers this. And regenerate icons isn't really going to help.

Comment (Super3222) [inline code/game/gamemodes/changeling/powers/horrorform.dm]: And set_species() already calls those.

Comment (ShadowDeath6) [inline code/game/gamemodes/changeling/powers/horrorform.dm]: just messing with those for trial and error with bugs, not planning to keep them if they won't work

**Comment by ShadowDeath6** [comment]
suffering

**Comment by ShadowDeath6** [comment]
I doubt I'll be working on this for a while, just gonna close till I manage to fix shit

**Pull request closed without merging**